<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MianWidget</class>
 <widget class="QWidget" name="MianWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MianWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>136</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color:transparent;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_About">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::FormatJustifyLeft"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Skin">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
   <property name="text">
    <string>搜索歌曲...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Search">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Search">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
   <property name="text">
    <string>搜索...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>191</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>搜索歌曲</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Lyric">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>150</y>
     <width>331</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_History">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>211</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>历史记录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>751</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_Front">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>上一曲</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Playerpause">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;播放|暂停&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Next">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下一曲&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Loopplay">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>循环播放</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgba(255, 255, 255, 50);  /* 设置背景为白色半透明 */
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSlider" name="VopSlider">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Vop">
      <property name="text">
       <string>0：00</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButton_Front</zorder>
   <zorder>pushButton_Playerpause</zorder>
   <zorder>pushButton_Next</zorder>
   <zorder>pushButton_Loopplay</zorder>
   <zorder>label_Vop</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer_2</zorder>
   <zorder>horizontalSpacer_3</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>horizontalSpacer_5</zorder>
   <zorder>horizontalSpacer_6</zorder>
   <zorder>VopSlider</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>761</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSlider" name="progressSlider">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Time">
      <property name="text">
       <string>00：00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clearHistory">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_Name</zorder>
  <zorder>lineEdit_Search</zorder>
  <zorder>pushButton_Search</zorder>
  <zorder>label</zorder>
  <zorder>textBrowser_Lyric</zorder>
  <zorder>listWidget_Search</zorder>
  <zorder>listWidget_History</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>clearHistory</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
